es-design-system/pages/organisms/es-file-upload.vue

```diff
@@ -0,0 +1,174 @@
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Default
+                        </dt>
+                        <dd>
+                            <code>{{ prop.default }}</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Description
+                        </dt>
+                        <dd>
+                            {{ prop.description }}
+                        </dd>
+                    </dl>
+                </ds-responsive-table-row>
+            </ds-responsive-table>
+        </div>
+        <div class="mb-450">
+            <h2>
+                EsFileInput event listeners
+            </h2>
+            <ds-responsive-table>
+                <ds-responsive-table-row
+                    v-for="eventListener in fileUploadEventListeners"
+                    :key="eventListener.name">
+                    <dl>
+                        <dt>
+                            Name
+                        </dt>
+                        <dd>
+                            <code>{{ eventListener.name }}</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Payload
+                        </dt>
+                        <dd>
+                            <code>{{ eventListener.payload }}</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Description
+                        </dt>
+                        <dd>
+                            {{ eventListener.description }}
+                        </dd>
+                    </dl>
+                </ds-responsive-table-row>
+            </ds-responsive-table>
+        </div>
+        <ds-doc-source
+            :comp-code="compCode"
+            comp-source="es-vue-base/src/lib-components/EsFileInput.vue"
+            :doc-code="docCode"
+            doc-source="es-design-system/pages/molecules/es-file-input.vue" />
+    </div>
+</template>
+<script>
+export default {
+    name: 'EsFileUploadDocs',
+    data() {
+        return {
+            compCode: '',
+            docCode: '',
+            uploadUrls: [],
+            fileUploadProps: [{
+                name: 'uploadUrls',
+                default: '[]',
+                description: 'A list of URLs to upload files to. When the number of files ready to upload matches '
+                + 'the number of URLs, the component will begin uploading.',
+            },
+            {
+                name: 'fileTypes',
+                default: 'None',
+                description: 'An array of accepted mime types for a file. If no argument passed, all file types are '
+                + 'accepted. These mime types follow the IANA Media Types.',
+            },
```

Comment:
nit: looks like `collapsed` and `maxFileSizeMb` didn't make it into the table?

Reply:
Correct! My mistake, although it seems like `collapsed` isn't being used as a prop for `EsFileUpload`, and instead collapses if there are files present. I will make it so it does not collapse on mobile.

Reply:
>  I will make it so it does not collapse on mobile

This is actually going to require a very disgusting way of calculating window size since it will be inside a Vue script and not CSS. I will shelve this idea for the time being.